tests/TestCase/Error/ErrorTrapTest.php

```diff
@@ -104,7 +104,8 @@ public static function logLevelProvider(): array
             // PHP error level, expected log level
             [E_USER_WARNING, 'warning'],
             [E_USER_NOTICE, 'notice'],
-            [E_USER_DEPRECATED, 'notice'],
+            // Log level is notice on windows because windows log levels are different.
```

Comment:
I will never understand windows...

Reply:
:shrug: They made divergent choices in the 80's that are still part of software all over the world.